init-softlayer.html.md.erb

```diff
@@ -63,83 +68,143 @@ jobs:
   - {name: blobstore, release: bosh}
   - {name: director, release: bosh}
   - {name: health_monitor, release: bosh}
+  - {name: powerdns, release: bosh}
   - {name: softlayer_cpi, release: bosh-softlayer-cpi}
 
   resource_pool: vms
   persistent_disk_pool: disks
 
   networks:
-  - {name: private, static_ips: [10.0.0.6]}
+  - name: default
 
   properties:
     nats:
-      address: 127.0.0.1
       user: nats
       password: nats-password
-
-    postgres: &db
-      listen_address: 127.0.0.1
-      host: 127.0.0.1
+      auth_timeout: 3
```

Comment:
The default value for auth_timeout is 1. I've removed some other default values.